javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+var countdownMessage = document.querySelector('.countdown-display');
+
+var count = 4;
+
+function countdown() {
+  if (count > 0) {
+    countdownMessage.textContent = count--;
+  } else {
+    clearInterval(setInterval(countdown, 1000));
+    countdownMessage.textContent = '~Earth Beeeelooowww Us~';
+  }
+}
+
+setInterval(countdown, 1000);
```

Comment:
Each call of `setInterval` returns a different interval id. Make sure you're clearing the original interval.